apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -282,6 +294,20 @@ static void tc_task_task_restart(void)
 	int ret_chk;
 	unsigned int remain;
 	g_callback = 0;
+
+	/* Check for NULL pid parameter  */
+
+	pid = task_create("tc_task_re", SCHED_PRIORITY_MAX - 1, 1024, restart_task, (char * const *)NULL);
+	TC_ASSERT_GT("task_create", pid, 0);
+
+	g_callback = 0;
+	ret_chk = task_restart(0);
+	TC_ASSERT_EQ("task_restart", ret_chk, ERROR);
+	TC_ASSERT_EQ("task_restart", errno, ENOSYS);
+	TC_ASSERT_EQ("task_restart", g_callback, 0);
+
+	/* Check for reinitialization of task using task_restart */
+
```

Comment:
You didn't add initialization of global variable. But you wrote "modified as you suggested". What is your opinion?

Reply:
Hi , i misunderstood with initializing before this TC, 
I will add it after this:
        ret_chk = task_restart(0);
        TC_ASSERT_EQ("task_restart", ret_chk, ERROR);
        TC_ASSERT_EQ("task_restart", errno, ENOSYS);
        TC_ASSERT_EQ("task_restart", g_callback, 0);
        g_callback = 0;